File: 2025-08/2025-08-14/space_jam.py
def space_jam(s):
    import re
    
    return re.sub(r"(.)", "\g<1>  ", s.replace(" ", "")).upper()[:-2]

print(space_jam("freeCodeCamp"))
print(space_jam("   free   Code   Camp   "))
print(space_jam("Hello World?!"))
print(space_jam("C@t$ & D0g$"))
print(space_jam("allyourbase"))

File: 2025-08/2025-08-14/spaceJam.js
function spaceJam(s) {
  s = s.replaceAll(" ", "").replace(/(.)/g, "$1  ")

  return s.slice(0, s.length - 2).toUpperCase();
}

console.log(spaceJam("freeCodeCamp"));
console.log(spaceJam("   free   Code   Camp   "));
console.log(spaceJam("Hello World?!"));
console.log(spaceJam("C@t$ & D0g$"));
console.log(spaceJam("allyourbase"));